examples/StandardFirmata/StandardFirmata.ino

```diff
@@ -805,6 +805,8 @@ void loop()
       if (IS_PIN_ANALOG(pin) && Firmata.getPinMode(pin) == PIN_MODE_ANALOG) {
         analogPin = PIN_TO_ANALOG(pin);
         if (analogInputsToReport & (1 << analogPin)) {
+          // Extra analogRead() to cover ADC cooling-off period for higher accuracy
+          analogRead(analogPin);
```

Comment:
This is a waste of cycles if a user only has one analog input. However you could add a counter that increments each time you enter this block and then only if that count > 0 perform the extra analogRead:

```c++
if (analogReadCounter++ > 0) {
  analogRead(analogPin);
}
```

and reset analogReadCounter to 0 before the for loop.